src/api-docs/source/index.html.md

```diff
@@ -123,8 +136,26 @@ along with its index in that transaction.
 Address indexer: enabled by `--index-address=true`. Allows lookup of all transactions
 involving a certain address.
 
+## Wallet plugin
+
+The main purpose of hsd node is to store blockchain data and to exchange messages with other nodes, for example
+messages about new blocks.
+
+Strictly speaking, any additional functionality is not part of the node itself. So the wallet is implemented as hsd node plugin. 
+
+By default hsd runs with wallet plugin activated, but it can be disabled by providing command line flag <code>--no-wallet</code>
+<br>(<code>no-wallet: true</code> in [hsd.conf](#hsd-conf-and-hsw-conf-files)).
```

Comment:
mentioned earlier, command-line only

Reply:
Added. So it seems it's not possible to run hsd without wallet from the config file, right?